doc/tests/sequence.rst

```diff
@@ -0,0 +1,13 @@
+``sequence``
+============
+
+``sequence`` checks if a variable is a sequence:
+
+.. code-block:: twig
+
+    {# evaluates to true if the users variable is a sequence #}
+    {% if users is sequence %}
+        {% for user in users %}
+            Hello {{ user }}!
+        {% endfor %}
+    {% endif %}
```

Comment:
Maybe we can a note about how we define a sequence?

Reply:
Done in https://github.com/twigphp/Twig/pull/3859/commits/9ecaf164fcf9de0a2f782153dbfa235d88a4efb6